pkg/webhook/server/certificate/manager.go

```diff
@@ -117,7 +117,7 @@ func (m *Manager) Start(stopCh <-chan struct{}) error {
 // of generating new one
 func (m *Manager) loadCACertFromCABundle() {
 	caBundle, err := m.CABundle()
-	if err != nil || len(caBundle) == 0 {
+	if err != nil {
```

Comment:
Hey what will happen if cert-manager is restarted and the current CA is dummy or empty cert,
will it trigger new cert creation?
@RamLavi @qinqon

Reply:
I think so yes. is that a problem?